README.md

```diff
@@ -1,18 +1,109 @@
 # libgc
 
-libgc is a garbage collector for Rust. It works by providing a garbage-collected
-`Gc<T>` smart pointer in the style of `Rc<T>`.
+[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://bors.tech)
 
-# Structure
+_libgc_ is a garbage collection library for Rust. It can be used as a standalone
+library, but it is highly recommended that programs are compiled with the
+companion [rustc fork](https://github.com/softdevteam/rustgc), which offers
+language support for much better performance.
+
+_libgc_ is in active development - there will be bugs!
+
+## Example
+
+_libgc_ provides a smart pointer, `Gc<T>`, which can be used to make
+garbage-collected values.
+
+```rust
+use libgc::{Gc, GcAllocator};
+
+#[global_allocator]
+static ALLOCATOR: GcAllocator = GcAllocator;
+
+
+fn foo() -> Gc<Vec<usize>> {
+    let foo = Gc::new(vec![1,2,3]);  
+    let a = foo; // GC pointers are copyable
+    let b = foo;
+
+    foo 
+}
+
+fn main() {
+    let gc = foo();
+}
+```
+
+## Overview
+
+If you want to write code with shared ownership in Rust, `Rc` makes this
+possible. Unfortunately, managing cyclic data structures with reference counting
+is hard: weak pointers are needed to break strong cycles and thus prevent memory
+leaks. In programs where these sorts of structures are common,
+garbage-collection is a natural fit. 
```

Comment:
I don't think one normally hyphenates "garbage collection"?

Reply:
Yeah, that should have been removed.

Reply:
Fixed: 2c3f464bab1f1dab1d543e2618347dbfced84e4e